backend/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+# Create your models here.
+
+class User(models.Model):
+    userId = models.IntegerField(null=False, blank=False)
+    passwordMd5hash = models.CharField(max_length=300, null=True, blank=True)
+    name = models.CharField(max_length=100, null=False, blank=False)
+    phoneNumber = models.IntegerField(null=False, blank=False)
```

Comment:
I think, we should make phone number string typed. Now, i`ll update the openapi branch, pull it to get this change